### PR TITLE
Small upgrades on the terminal

### PR DESCRIPTION
**Pull request by ketodin**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (exdeejay) [inline LICENSE:1]: Remove please

File: README.md [hunk 2]
```diff
@@ -1,31 +1,10 @@
 # motion-canvas-components
 
-A component library for [Motion Canvas](https://motion-canvas.github.io/). Feel free to fork and/or PR!
+A terminal component for [Motion Canvas](https://motion-canvas.github.io/).
```
Comment (exdeejay) [inline README.md:3]: Please revert this file, this repo's intended to be a general component library (even if there's only one component)

File: package.json [hunk 3]
```diff
@@ -1,15 +1,28 @@
 {
-  "name": "motion-canvas-components",
-  "version": "0.1.0",
-  "main": "./src/index.ts",
+  "name": "@dct-lulu/motion-canvas-terminal",
```
Comment (exdeejay) [inline package.json:2]: Please revert all but the dependency versions

File: src/index.ts [hunk 4]
```diff
@@ -1,2 +1 @@
-export * from './Polygon';
```
Comment (exdeejay) [inline src/index.ts:1]: Please revert

**Review by exdeejay — CHANGES_REQUESTED** [review]
Appreciate the PR, and I understand you're coming from your own repo's main branch, but please format this PR to keep the original scope of this repo in mind.

**Comment by ketodin** [comment]
I'm currently trying to update the terminal component to be usable in the latest version of MC. I'm kind of new to using GitHub, but I think it would be better if there would be a package on npm to easily use the component also wouldn't be better if there is a Github repo per components?

**Comment by exdeejay** [comment]
You're free to take the full source code for this component and make it a standalone npm package/repo on your own, but this repository specifically is not supposed to be a public npm package nor a single component.

**Pull request closed without merging**